ui/app/pages/home/index.scss

```diff
@@ -36,4 +36,70 @@
   &__tab {
     flex-grow: 1;
   }
+
+  &__connect-status-text {
+    display: flex;
+    flex-direction: column;
+    @extend %content-text;
+    padding-left: 24px;
+    padding-right: 24px;
+
+    div {
+      &:last-child {
+        margin-top: 26px;
+      }
+    }
+  }
+
+  &__connected-status-popover {
+    width: 329px;
+    height: 295px;
+    margin-top: -12px;
+
+    .popover-header {
+      padding-bottom: 4px;
+    }
+
+    .popover-content {
+      overflow-y: auto;
+    }
+  }
+
+  &__connected-status-popover-bg {
+    height: 34px;
+    width: 96px;
+    border-radius: 34px;
+    position: absolute;
+    top: 82px;
+    left: 18px;
```

Comment:
The outline for the button looked a bit.... squished? I tried adjusting the position and width, and it came out looking a bit better I think

```suggestion
    width: 110px;
    border-radius: 34px;
    position: absolute;
    top: 82px;
    left: 12px;
```

<details>
<summary>Screenshots:</summary>

Before:

![before](https://user-images.githubusercontent.com/2459287/78716158-d9d21a80-78f4-11ea-8879-a7b1069d3fa0.png)

After:

![after](https://user-images.githubusercontent.com/2459287/78716168-dccd0b00-78f4-11ea-8efb-aad1db205ab4.png)

</details>

Reply:
Change applied in 593fdffa2